ledger/ledger-api-common/BUILD.bazel

```diff
@@ -47,7 +47,6 @@ da_scala_library(
         "@maven//:io_dropwizard_metrics_metrics_core",
         "@maven//:io_grpc_grpc_netty",
         "@maven//:io_netty_netty_handler",
-        "@maven//:io_zipkin_brave_brave",
```

Comment:
If we don't use it anywhere else, we can probably remove it from `//bazel-java-deps.bzl` as well.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/10256/commits/751b11ec854d156409c0bb05de84f54c12103294 🙇🏻